MUXSDKKaltura/MUXSDKKalturaTests/MUXSDKStatsTests.swift

```diff
@@ -172,4 +172,118 @@ class MUXSDKStatsTests: XCTestCase {
         )
         self.assertDispatchedCustomerViewerDataEventsAtIndex(index: 0, expectedCustomerViewerData: expectedViewerData)
     }
+    
+    func testDestroyPlayer() {
+        guard
+            let customerData = MockedData.customerData
+        else {
+            XCTFail("Customer data not found")
+            return
+        }
+        
+        MUXSDKStats.monitorPlayer(player: MockedData.player, playerName: MockedData.playerName, customerData: customerData)
+        MUXSDKStats.destroyPlayer(name: MockedData.playerName)
+        
+        let expectedEvents = [
+            MUXSDKPlaybackEventViewInitEventType,
+            MUXSDKDataEventType,
+            MUXSDKPlaybackEventPlayerReadyEventType,
+            MUXSDKPlaybackEventViewEndEventType
+        ]
+        
+        self.assertDispatchedEventTypesMatch(
+            expectedEventTypes: expectedEvents,
+            for: MockedData.playerName
+        )
+        
+        MUXSDKStats.destroyPlayer(name: MockedData.playerName)
```

Comment:
This can be a little confusing for someone new to the code might be good to actually document the test process like:
// start monitor player to emit events 
// assert expected events
// clean player instance after test